src/cmd-build

```diff
@@ -372,10 +372,7 @@ if [ "${commit}" == "${previous_commit}" ] && \
     fi
 else
     ostree init --repo=repo --mode=archive
-    ostree pull-local --repo=repo "${tmprepo}" "${ref}"
-    if [ -n "${ref_is_temp}" ]; then
-        ostree refs --repo=repo --delete "${ref}"
-    fi
+    ostree pull-local --repo=repo "${tmprepo}" "${buildid}"
```

Comment:
Hmm, doesn't this mean there won't be the real ref at all in FCOS tarballs? We *might* have code which relies on that. Maybe just

```suggestion
    ostree pull-local --repo=repo "${tmprepo}" "${buildid}" ${ref}
```

so it creates the ref too if it's not empty?

Reply:
OK, done.